scss/pages/blog.scss

```diff
@@ -43,64 +45,259 @@ body.blog {
+    span {
+      margin: 0;
+    }
+  }
+   
+  @media (min-width: 1001px) {
+    .post-card-large {
+      grid-column: 1 / span 3;
+      display: grid;
+      grid-gap: 4vmin;
+      grid-template-columns: 1fr 1fr 1fr;
+      min-height: 280px;
+
+      .title {
+        margin-top: 0;
+        font-size: 3.2rem;
       }
-      
-      .body {
-        @extend .cooked;
-        
-        pre code {
-          overflow: auto;
-          -moz-tab-size: 4;
-          tab-size: 4;
-          
-          &.lang-markdown {
-            white-space: pre-wrap;
-          }
+
+      .excerpt p {
+        margin-bottom: 1.5em;
+        font-size: 1.7rem;
+        line-height: 1.55em;
+        -webkit-line-clamp: 8;
+      }
+
+      .topic-details {
+        justify-content: center;
+      }
+
+      .topic-image {
+        position: relative;
+        grid-column: 1 / span 2;
+        margin-bottom: 0;
+        min-height: 380px;
+        img {
+          position: absolute;
+          width: 100%;
+          height: 100%;
         }
       }
     }
+    .post-card-large:not(.no-image) .header {
+      margin-top: 0;
+    }
+  }
+  .user-top img {
+    margin-right: 6px;
+  }
+  @media (max-width: 500px) {
+    .title {
+      font-size: 1.9rem;
+    }
+    .excerpt {
+      font-size: 1.6rem;
+    }
+  }
+}
+
+.subscribe {
+  position: fixed;
+  z-index: 2;
+  bottom: 20px;
+  display: flex;
+  align-items: center;
+  right: 12px;
+  border-radius: 2px;
+  background: #fdf8f5;
+  padding: 4px;
+  cursor: pointer;
+
+  .subscribe-avatar {
+    border-radius: 50%;
+    vertical-align: sub;
+  }
+  .subscribe-label {
+    float: right;
+    padding: 10px 0 0 5px;
+    font-size: 18px;
+  }
+}
+
+.subscribe-modal {
+  visibility: hidden;
+
+  &.active {
+    visibility: visible;
+    z-index: 999;
+    padding-top: 300px;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    position: fixed;
+    background-color: rgba(0,0,0,0.4);
+  }
+
+  .subscribe-modal-content {
+    width: 300px;
+    height: 200px;
+    background: #fefefe;
+    padding: 1px 14px 14px 14px;
+    position: relative;
+    margin: 0 auto;
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2),0 6px 20px 0 rgba(0,0,0,0.19);
+    -webkit-animation-name: animatetop;
+    -webkit-animation-duration: 0.4s;
+    animation-name: animatetop;
+    animation-duration: 0.4s;
+    border-radius: 10px;
+
+    .subscribe-modal-close {
+      float: right;
+      font-size: 28px;
+      font-weight: bold;
+      &:hover,
+      &:focus {
+        text-decoration: none;
+        cursor: pointer;
+      }
+    }
+    .checkbox {
+      float: right;
+      height: 20px;
+      width: 20px;
+    }
+    .subscribe-buttons {
+      padding-top: 20px;
+      border-top: 1px solid #e9e9e9;
+    }
+    .subscribe-checkbox {
+      margin-bottom: 20px;
+      margin-top: 20px;
+    }
+    .modal-header {
+      height: 25%;
+      border-bottom: 1px solid #e9e9e9;
+    }
   }
```

Comment:
@MeghnaAJ As our model-contents is a flexbox, let's use ``flex: 1;`` on the ``model-content``.